app/lib/foreman/http_proxy.rb

```diff
@@ -1,5 +1,5 @@
 module Foreman
-  module HTTPProxy
+  module HttpProxy
```

Comment:
Perhaps HTTP needs to be its own acronym as well

Reply:
Could you then update https://github.com/theforeman/foreman/pull/10076? For some reason though, I didn't make this an acronym in my alternative. Maybe there are more pitfalls than it seems...